src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --primary-color: rgba(0, 172, 220, 1);
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, system-ui;
+}
+
+.header {
+  /* border: 1px solid red; */
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header img {
+  display: block;
+}
+
+.nav-bar {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav-bar li {
+  /* border: 1px solid green; */
+  list-style: none;
```

Comment:
Avoid using tag names for styling. It's better to style using classes to avoid affecting list styles globally. Consider adding a class to these 'li' elements and style them using that class.